### ISSUE TITLE
Cannot add a new form status

### ISSUE THREAD

**Issue by 1stevengrant** [solspace/craft-freeform#451]
When I try to add a custom form status from the control panel in production and locally, it errors out on the following

```
Solspace\Freeform\Controllers\StatusesController::getNewOrExistingStatus(): Argument #1 ($statusId) must be of type int, null given, called in /Users/xxx/Sites/xxx/vendor/solspace/craft-freeform/packages/plugin/src/Controllers/StatusesController.php on line 65
```

**Steps to reproduce**

1. go to the control panel
2. try to add a new form status

**Expected behavior**

It should create the new status

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.2.3
- Freeform Version: 4.0.2
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Additional context**
We have multi-site configured

**Comment by kjmartens** [comment]
Sorry for the delay and the trouble, @1stevengrant.

I can duplicate this issue as well. We'll make a fix for this shortly. 🙂

**Comment by kjmartens** [comment]
This has now been resolved in Freeform 4.0.4. 🙂
